### PR TITLE
Admin Actuator

### PR DESCRIPTION
**Pull request by joseadp**
Features of this version:
* Encrypt / Decrypt message
* Operations in the database:
-- Insert resource
-- Insert actuator
-- Insert actions

### CODE REVIEW

File: development/TMA_Admin_Console/.project [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (nmsa) [inline development/TMA_Admin_Console/.project]: eclipse-specific

File: development/TMA_Admin_Console/.settings/org.eclipse.jdt.core.prefs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+eclipse.preferences.version=1
```
Comment (nmsa) [inline development/TMA_Admin_Console/.settings/org.eclipse.jdt.core.prefs]: eclipse-specific.

File: development/TMA_Admin_Console/.settings/org.eclipse.m2e.core.prefs [hunk 3]
```diff
@@ -0,0 +1,4 @@
+activeProfiles=
```
Comment (nmsa) [inline development/TMA_Admin_Console/.settings/org.eclipse.m2e.core.prefs]: eclipse-specific.

**Comment by joseadp** [comment]
@nmsa , I remove the files. Can you review it again?

**Comment by nmsa** [comment]
> @nmsa , I remove the files. Can you review it again?

Great, thanks!

**Pull request merged**